docs/doc_env.yaml

```diff
@@ -0,0 +1,29 @@
+name: documentation
+channels:
```

Comment:
Do we need to be able to install `itctools` for the dos to build? If so, we need this to at least contain the `devtools/conda-envs/test_env.yaml` prereqs too.

Reply:
The molssi tutorial suggests removing anything that isn't needed to install the repo, including the testing packages. The documentation builds fine right now - i.e. pytest is only imported when pytest is run, not on itctools import. Does that make sense? I can make the envs identical if you want, though.